tools/db_bench_tool.cc

```diff
@@ -1124,6 +1124,10 @@ DEFINE_bool(rate_limit_auto_wal_flush, false,
             "false) after the user "
             "write operation");
 
+DEFINE_bool(async_io, false,
```

Comment:
How about "async_read"? iiuc, this only covers Get/MultiGet?

Reply:
Anand suggested to use `async_io` instead of `async_read` as he mentioned we might use this for other async io as well in future.

Reply:
Would it make sense to use a separate flag for that case? But since Anand has more context on this, it's totally OK with me.

Reply:
Once he get back it will follow up with him on this.